docs/examples.md

```diff
@@ -228,7 +228,7 @@ For the common case where you want to [`include`](attrs.filters.include) or [`ex
 
 >>> asdict(
 ...     User("jane", "s33kred", 42),
-...     filter=filters.exclude(fields(User).password, int))
+...     filter=filters.exclude(fields(User).password, "password", int))
```

Comment:
This makes no sense, since it's using password twice which is confusing (at least it's confusing me right now).

Maybe add a field?

Reply:
Maybe `email`?